sched/sched/sched_timerexpiration.c

```diff
@@ -233,15 +212,6 @@ static uint32_t nxsched_cpu_scheduler(int cpu, uint32_t ticks,
 
   /* Returning zero means that there is no interesting event to be timed */
 
-#ifdef KEEP_ALIVE_HACK
```

Comment:
Just for reference: https://github.com/apache/incubator-nuttx/commit/6546fa39c7a#diff-ec549151b9b686bd3d14a18d3a6a0ca940987bff9b59ab299166ea506f604219